internal/cli/command/login/oidc.go

```diff
@@ -77,284 +35,11 @@ func runServer(banzaiCli cli.Cli, pipelineBasePath string) (string, error) {
 	}
 	issuerURL.Path = "/dex"
```

Comment:
We could get the issuerURL more correctly here with calling `/auth/dex/login` on `pipelineBasePath` and check the redirect `Location` header.

Reply:
cool thx, i added this

Reply:
great!